static/app/views/issueDetails/utils.tsx

```diff
@@ -273,13 +274,13 @@ export function getGroupEventDetailsQueryData({
 
 export function useHasStreamlinedUI() {
   const location = useLocation();
-  const organization = useOrganization();
+  const user = useUser();
   if (location.query.streamline === '0') {
     return false;
   }
+
   return (
-    location.query.streamline === '1' ||
-    organization.features.includes('issue-details-streamline')
+    location.query.streamline === '1' || !!user?.options?.prefersIssueDetailsStreamlinedUI
```

Comment:
could probably remove the query param stuff if there's a button

Reply:
That makes sense, though I'd prefer to leave it in for now because of one case.

For testing on issues from orgs without the flag, to access the streamlined UI, you'd have to open an issue from an org with it, and enable the UI to set your user preference, then refresh the issue you were looking at, rather than just changing the UI.

That and it's useful for sharing links to the pages while its in development.

Reply:
I guess one way around this is for us to have the button conditional on if the user has set the user option at all, not it's value, since that implies they've accessed the new UI before, but for now I think it's okay to keep the query param.